query-node/schemas/membership.graphql

```diff
@@ -119,6 +119,8 @@ type Membership @entity {
 
   "NFTs owned by this member"
   ownedNfts: [OwnedNft!]! @derivedFrom(field: "ownerMember")
+
+  #nftCollectorInChannels: [Channel!]! @derivedFrom(field: "nftCollectors")
```

Comment:
Should this be commented out?

Reply:
These commented lines have been removed, and proper relationship has been created here.